### ISSUE TITLE
24-hour Stomach Food Rule inconsistent with Single Player Game Style

### ISSUE THREAD

**Issue by Wally1169** [StrangeLoopGames/EcoSuggestions#489]
In the Single Player game style, the Server starts and stops with the Loading of the game so components like the food staying in the stomach for 24 hours and affecting skill point calculations doesn't really fit that model.  Perhaps using the SkillGainMultiplier (but dividing instead of multiplying) would be a fair method of adjusting this component.

**Comment by PacifyerGrey** [comment]
Isn't SkillGainMultiplier 24 in a single game? That would mean stomach would empty in 1 hour? Most of actions in single player take the same time from a player and this would probably result in player getting empty stomach a little too often if he is not doing some calorie heavy job.

However I do generally agree with you that some processes should be accelerated in single player.
This and plant growth times (farming and especially tree seeds) should be effected.

**Comment by Wally1169** [comment]
That’s true.  Most of the time I’m playing in single player, though, I’m at least doing enough mining or logging or building that I need to eat.

At least that latter stuff (growth times) is editable in the ecosim file - although the trees, at least, are listed in days, so unless the field takes a float value less than 1, you’re limited to a minimum of 1 day of played time.
